spec/components/procedures/card/champs_component_spec.rb

```diff
@@ -13,16 +13,23 @@
     end
 
     context 'when errors on types_de_champs_public' do
-      let(:types_de_champ_public) { [{ type: :drop_down_list, options: [] }] }
+      let(:types_de_champ_public) { [{ type: :drop_down_list }] }
```

Comment:
idem, utiliser une autre source d'erreur, tu peux prendre les repetitions sans enfant

```suggestion
      let(:types_de_champ_public) { [{ type: :repetition, children: [] }] }
```